rep-0003.rst

```diff
@@ -273,7 +273,7 @@ Requirements:
 | Qt5     |     5.5.1     |     5.6.1      |    5.7.1     |    5.7.1       |   5.7.1   |
 +---------+---------------+----------------+--------------+----------------+-----------+
 
-" * " means that this is not the upstream version but a package distributed by OSRF or the community.
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
```

Comment:
yeah I agree that it's a non trivial thing to explain to new comers. If you come up with a better wording in the future feel free to comment here and I'll happily iterate on this. I merged it as is so that the rep reflects the actual dependencies that Lunar shipped with since it's been released.
Thanks for the feedback!